docs/docs/basic-hardware-software-requirements.md

```diff
@@ -2,6 +2,10 @@
 title: Basic Hardware and Software Requirements
 ---
 
+Gatsby requires at least 1GB of RAM.
+
+------
+
 This is a stub. Help our community expand it.
 
-Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your pull request gets accepted.
+Please use the [Gatsby Style Guide](/docs/contributing/gatsby-style-guide.md) to ensure your pull request gets accepted.
```

Comment:
The docs are supposed to be read on gatsbyjs.org so please revert this change :)